providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -690,6 +704,19 @@ static int ec_gen_set_template(void *genctx, void *templ)
     return ec_gen_set_group(gctx, EC_GROUP_get_curve_name(ec_group));
 }
 
+static int ec_param_encoding_name2id(const OSSL_PARAM *p, int *id)
+{
+    if (p->data_type != OSSL_PARAM_UTF8_STRING)
```

Comment:
OSSL_PARAM_UTF8_STRING_PTR also makes sense as a possibility.

(because you love the PTR types)

Reply:
Unless it is really needed I am not going to support it...